packages/facts-base/facts_base_server.js

```diff
@@ -56,7 +56,7 @@ Facts.resetServerFacts = function () {
 // called?
 Meteor.defer(function () {
   // XXX Also publish facts-by-package.
-  Meteor.publish(FACTS_PUBLICATION, function () {
+  Meteor.publish(FACTS_PUBLICATION, async  function () {
```

Comment:
It'd be nice to document the fact that publications can be async - it's only in the changelog now.